Sources/Datadog/FeaturesIntegration/WebView/WKUserContentController+Datadog.swift

```diff
@@ -16,10 +16,32 @@ public extension WKUserContentController {
     internal func __addDatadogMessageHandler(allowedWebViewHosts: Set<String>, hostsSanitizer: HostsSanitizing) {
         let bridgeName = DatadogMessageHandler.name
 
+        let contextProvider = (Global.rum as? RUMMonitor)?.contextProvider
+
+        var logEventConsumer: WebLogEventConsumer? = nil
+        if let loggingFeature = LoggingFeature.instance {
+            logEventConsumer = WebLogEventConsumer(
+                userLogsWriter: loggingFeature.storage.writer,
```

Comment:
`loggingFeature.storage.writer` is a [consent-aware data writer](https://github.com/DataDog/dd-sdk-ios/blob/fc693d3089d93ba3bc8e5d3ef21e841b6e973125/Sources/Datadog/Core/Feature.swift#L91). Do we need to consider tracking consent for events received from Browser SDK? Shouldn't we assume that all these were already authorized in Browser SDK and we can bypass consent consideration in native SDK?

Reply:
you may want to discuss with @mariusc83 

as we discussed today, the only tricky situation is consent is given in browser-sdk but not in native sdk as then the browser events will be lost. but i personally see this scenario as an edge case or one in which the developer should take consent in browser-sdk and forward it to native sdk.
in short, i think it makes sense to use `context-aware data writer` here

Reply:
@mariusc83 what's our take on this? In any case, we should avoid inconsistency here - either by relying only on Browser SDK consent (when we receive an event, we know it was already authorized) or by adding additional logs to inform the user on inconsistency (that Browser SDK events will be lost).

Reply:
@bcaudan told me `browser-sdk` doesn't have tracking consent feature and it makes sense to go with the consent from native SDK as the events will eventually be shown in the native RUM session